dts/bindings/timer/nxp,mcux-gpt.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: NXP MCUX General Purpose Timer
+version: 0.1
+
+description: >
+    This is a representation of the NXP MCUX General Purpose Timer (GPT)
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nxp,imx-gpt"
```

Comment:
Please rename the file to match the constraint. Also, many of these properties can now be removed, see #16693

Reply:
thanks, done